crates/ecolor/src/rgba.rs

```diff
@@ -219,6 +219,11 @@ impl Rgba {
             linear_u8_from_linear_f32(a.abs()),
         ]
     }
+
+    /// Blend two colors, so that `self` is behind the argument.
+    pub fn blend(self, on_top: Self) -> Self {
```

Comment:
It's technically redundant, but I think this should clarify that since `Rgba` is a linear color this performs blending in linear, as opposed to what `Color32` does.